crates/engine/tree/src/tree/payload_processor/multiproof.rs

```diff
@@ -1458,6 +1458,10 @@ impl MultiProofTask {
 /// Context for multiproof message batching loop.
 ///
 /// Contains processing state that persists across loop iterations.
+///
+/// This context is used by `process_multiproof_message` to batch multiple messages of the same
+/// type together. Messages are received via `try_recv` in a loop, allowing consecutive messages
+/// of the same type to be accumulated and processed together for better efficiency.
```

Comment:
```suggestion
/// Used by `process_multiproof_message` to batch consecutive same-type messages received via
/// `try_recv` for efficient processing.
```

The current one might be too verbose, suggested a more concise one!